guidelines/data.md

```diff
@@ -0,0 +1,20 @@
+**_Journal of Cheminformatics_ will only publish research or software that is entirely reproducible by third parties.
+This means that any datasets, software and algorithms that are required to reach the conclusions stated in the
+paper must be provided as supplemental materials, or be otherwise accessible without the need for registration,
+login or agreement with license terms other than Creative Commons licenses for data and text and
+[OSI-approved Open Source Licenses](http://opensource.org/licenses/alphabetical)
+for software. For any software, the source code must be provided.**
+
+Data articles should describe a curated, CCZero-license dataset likely to be of broad utility. The dataset must
+be curated as detailed in this publication and archived for long-term reuse. A Data article describes the content
+of the data set, the method used to curate the data, and a description of how the dataset deposition complies with
+the FAIR principles. Unlike with a Database paper, a Data paper focuses on the data itself and not how the data
+can be (graphically) explored.
```

Comment:
I'd phrase is as "graphically or otherwise"

Reply:
Also, it could be useful to list some examples of datasets we would consider. For example, if somebody computes DRAGON descriptors for PubChem would we want it? Conversely, how indirect a connection to chemistry would we be willing to accept? I'm thinking proteomics datasets as a possible case

Reply:
> 
> 
> I'd phrase is as "graphically or otherwise"

Yes I got stuck on this phrasing too. Databases are not just about data exploration, right?